src/ChatCore.ts

```diff
@@ -16,14 +26,16 @@ import { ApiResponseValidator } from "./validation/ApiResponseValidator";
  */
 export class ChatCore {
   private chatConfig: ChatConfig;
-  private httpService: HttpService;
+  private httpService: Poster;
   private endpoints: Endpoints;
+  private nightly: boolean;
 
-  constructor(chatConfig: ChatConfig) {
+  constructor(chatConfig: ChatConfig, nightly?: boolean) {
```

Comment:
thoughts on making this directly take in all possible values for promptPackage instead? (we can add a new type `PromptPackage`)